### ISSUE TITLE
Gale 4th Animation Not Appearing

### ISSUE THREAD

**Issue by Cathy-Smiths-Biggest-Fan** [bunfan/beat-banger-public#347]
### Discord Tag

panthom

### Issue Description

In Gale's level, the 4th animation doesn't ever appear (gale_rev_cow.png in game files) even though the animation file is there. It also doesn't appear in the scene viewer.

### OS

Windows

### Version

3.531

### Modifiers Enabled?

None

### What level or section is the scene present.

Gale (I specifically experienced this on expert if that changes anything)

**Comment by FlappyBoi032** [comment]
had the same issue. Did her dirty

**Comment by hackerTiger21** [comment]
Did a little digging in the game files, the `keyframes.cfg` file for Gale doesn't have the necessary element in the `loops` array. I've modified the file that came with the game with the necessary array element, and I've verified that it works. I've attached the modified file, you just need to change the file extension to ".cfg" because GitHub would only accept a ".json" file extension for the upload.

[keyframes.json](https://github.com/user-attachments/files/15693745/keyframes.json)
